app/navigation/nonsipp/SharesNavigator.scala

```diff
@@ -161,7 +161,21 @@ object SharesNavigator extends JourneyNavigator {
       controllers.nonsipp.shares.routes.SharesIndependentValuationController.onPageLoad(srn, index, NormalMode)
 
     case SharesIndependentValuationPage(srn, index) =>
+      userAnswers.get(WhyDoesSchemeHoldSharesPage(srn, index)) match {
+        case Some(Acquisition) =>
+          controllers.routes.UnauthorisedController.onPageLoad()
```

Comment:
If this is for the conditional page, then within this `Acquisition` branch I think there also needs to be a check that `TypeOfShares` is `SponsoringEmployer`. And if it's `Unquoted` or `ConnectedParty`, then it navigates `SharesTotalIncomeController` instead.

Reply:
changing now